### PR TITLE
ENH: Make q2studio env activate instructions clear

### LINKED ISSUE #154: q2studio install instruction aren't clear about when to activate env
This came up [previously](https://github.com/qiime2/docs/issues/102), but still seems to be a [bit confusing](https://forum.qiime2.org/t/error-installing-q2studio/657). Perhaps we need to add an explicit `source activate qiime2-$RELEASE` at the top of the page? We could retool the [`note`](https://github.com/qiime2/docs/blame/master/source/interfaces/q2studio.rst#L33), or drop it altogether, I have no preference.

### PR DESCRIPTION
**Pull request by maxvonhippel**
Fixes #154

### CODE REVIEW

**Comment by maxvonhippel** [comment]
I think that readers may sometimes ignore `::note` blocks.  Therefore, making the note part of the instructions should hopefully make readers more likely to read it, especially when a command block is involved.  And by including a note about `if you get an error at this step ...` after that, we can (I reason) redirect anyone who does run into this problem back to the documentation they should have worked through prior to attempting to install the studio.

File: source/interfaces/q2studio.rst [hunk 1]
```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
 
 .. note:: QIIME 2 Studio currently requires a source installation. In the future, we will provide a package distribution/installer which will greatly simplify the installation process presented here. An experimental build of QIIME 2 Studio is available as a pre-installed package in the :doc:`VirtualBox image <../install/virtual/virtualbox>`.
 
+Before being able to successfully run QIIME 2 Studio, you must always confirm that your QIIME 2 ``conda`` environment is activated. Failing to activate your environment prior to starting QIIME 2 Studio will cause issues and most likely crash the application. If you have not activated your ``conda`` environment already, you should do so now:
```
Comment (thermokarst) [inline source/interfaces/q2studio.rst]: Nice! I have a few suggested tweaks, but overall I like the flow!

```
Launching QIIME 2 Studio requires an active ``conda`` environment
that has QIIME 2 installed in it. Failure to activate a compatible
``conda`` environment will likely produce errors, and possibly
cause the application to crash. If you have not activated your ``conda``
environment already, please do so now:
```

Feel free to tighten that up a bit if something jumps out at you!

File: source/interfaces/q2studio.rst [hunk 2]
```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
...
+
+.. command-block::
+	:no-exec:
```
Comment (thermokarst) [inline source/interfaces/q2studio.rst]: Can you please align the lines in this `command-block` with the `c` in `command`? This is the convention we have been trying to follow elsewhere in the docs (e.g. [here](https://github.com/qiime2/docs/pull/172/files#diff-fbd8ec22aadd68c37abf3f4de14d4355R43)). Thanks!

File: source/interfaces/q2studio.rst [hunk 3]
```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
...
+
+	source activate qiime2-2017.6
+
+If you run into an error at this point, then there is probably an issue with your install of QIIME 2. In that case, you should revisit the :doc:`install documentation <../install/index>` and make sure you have successfully installed QIIME 2 and all of its dependencies on your system.
```
Comment (thermokarst) [inline source/interfaces/q2studio.rst]: I'm not sure if we need this section if we include the edits suggested above.

```
Please note, your ``conda`` environment might have a
different name, if you or your system administrator
provided one during :doc:`installation of QIIME 2<../install/index>`.
```

Thoughts?

Comment (maxvonhippel) [inline source/interfaces/q2studio.rst]: I like your re-write, but I also feel that we might want to mention more than once that QIIME 2 *must* be installed in the `conda` environment.

How about:

````
Please note, your ``conda`` environment might have a different name, if you or your system administrator provided one during :doc:`installation of QIIME 2<../install/index>`. If you encounter errors unrelated to the environment name, you should confirm that QIIME 2 is installed in the environment.
````

> qiime info

Comment (maxvonhippel) [inline source/interfaces/q2studio.rst]: I'm happy with this.  Do you think we need to mention again that QIIME 2 **must** be installed in the environment, or have we covered that sufficiently?

Comment (maxvonhippel) [inline source/interfaces/q2studio.rst]:
````
Please note, your ``conda`` environment might have a 
name other than ``qiime2-2017.6``, if you or your system 
administrator provided one during 
:doc:`installation of QIIME 2<../install/index>`.
````

I think this way (by including the name explicitly) users who are not familiar with any command line usage will understand that `qiime2-2017.6` is the name of the environment, and not some black magic computer gizmo thing.

**Comment by maxvonhippel** [comment]
Ok, this is ready for review @thermokarst .  Thanks!

File: source/interfaces/q2studio.rst (hunk 3 above)
Comment (thermokarst) [inline source/interfaces/q2studio.rst]: > Do you think we need to mention again that QIIME 2 must be installed in the environment, or have we covered that sufficiently?

If you included the edits suggested above, this has already been covered.

```
Please note, your ``conda`` environment might have a 
...
```

👍 🏅

**Pull request merged**